challengeFive.js

```diff
@@ -0,0 +1,11 @@
+function hideNumber(number){
+  const cardNumber = number.toString();
+  const hiddenNumber = "*".repeat(cardNumber.length - 4)
+   return  hiddenNumber + cardNumber.slice(- 4) 
+ 
+}
+
+
+
+
```

Comment:
Generally, you have really ugly formatting issues happening. You need to style on a code formatter and linter. Then configure that with VS Code. This is not good!